app/views/content_items/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>List Content Items</h1>
+<h1><%= @organisation.slug %></h1>
```

Comment:
@pmanrubia Is it worth "cheating" for now and doing `.gsub('-', ' ').humanize` here, abstracting later to the model?